packages/cascara/src/ui/Pagination/Pagination.fixture.js

```diff
@@ -54,7 +54,7 @@ const WithStateHook = (fixtureProps) => {
       </p>
       <Pagination {...fixtureProps} state={paginationState} />
       <Pagination {...fixtureProps} state={paginationState} />
-      <JsonPlaceholder data={paginationState} isInitialOpen title='state' />
```

Comment:
I don't think we want to be commenting these out. Do do we recall the reason?

Reply:
Thank you so much for this comment. I commented just for a moment because of the error in eslint and then, I forgot to recover it back.